packages/dx-scheduler-core/src/types/index.ts

```diff
@@ -0,0 +1,12 @@
+export * from './scheduler-core.types';
+export * from './all-day-panel.types';
+export * from './appointment-form.types';
+export * from './appointment-tooltip.types';
+export * from './date-navigator.types';
+export * from './editing-state.types';
+export * from './horizontal-rect.types';
+export * from './month-view.types';
+export * from './vertical-rect.types';
+export * from './view-state.types';
+export * from './utils.types';
+export * from './week-view.types';
```

Comment:
No newline in the end of file

Reply:
Not only here, please check other files too

Reply:
I have a new line in the end of this file